### ISSUE TITLE
Can't import profanity_filter

### ISSUE THREAD

**Issue by vdarakjian** [rominf/profanity-filter#1]
I'm getting this error when trying to import the library on a python terminal. Using both python 3.6.0 and 3.7.0
![screenshot from 2018-10-21 17-35-38](https://user-images.githubusercontent.com/19177064/47272160-c93eb080-d557-11e8-9894-e0c23c313e48.png)

**Comment by rominf** [comment]
@vdarakjian, thank you for the report! Sorry.
That's a bug. Please wait till I add `requirements-git.txt` with updated requirements.

**Comment by rominf** [comment]
I fixed this and uploaded a new version to PyPI. Please try it and report is everything OK now.

**Comment by vdarakjian** [comment]
Hey! Sorry about the delay. Thank you!
I tried again and that bug is gone, but now I'm facing another problem. After importing the lib, when initializing the class I got the following error.

![profanity_filter_error](https://user-images.githubusercontent.com/19177064/48723372-9b5b9300-ec0d-11e8-9a29-9da393ff5c23.jpg)

I don't if it's helpful but i'm getting this with or without using the deep analysis.
